osier/equations.py

```diff
@@ -78,14 +80,113 @@ def get_dispatchable_names(technology_list):
         The list of dispatchable technology names.
     """
 
-    dispatchable_names = [t.technology_name for t in technology_list if t.dispatchable]
+    dispatchable_names = [
+        t.technology_name for t in technology_list if t.dispatchable]
 
     return dispatchable_names
 
 
+def per_unit_capacity(technology_list,
```

Comment:
It's not used, but the uniformity of objective function structures allow users to define their own objectives and pass them to osier. Otherwise, I would have to devise a way to check what kind of objective is being
passed (i.e. whether it needs a solved model or not) so that osier could give it the right information.

For concreteness, during a genetic algorithm simulation the following occurs

1. Osier generates a test population of technology capacities.
2. Those technologies are passed to the dispatch model which determines how much energy each one produces
3. It calculates the user-specified objectives.

```py
from osier import CapacityExpansion
from osier import total_cost, annual_emission
objectives_list = [total_cost, annual_emission]

problem = CapacityExpansion(technology_list,
    demand,
    objectives=objectives_list)
###############################3
# inside the model run
# generate capacities
# determine dispatch

for obj_func in self.objectives:
    obj = obj_func(technology_list=self.technology_list,
        solved_dispatch_model=self.dispatch_model)
    out['F'].append(obj)
```

For more context, look at [CapacityExpansion lines 176-188](https://github.com/arfc/osier/blob/da88fd6418afed238b467fd918f8f425e8e58144/osier/models/capacity_expansion.py#L176-L188).